### ISSUE TITLE
Issue with Vert.x 3.8.1

### ISSUE THREAD

**Issue by cescoffier** [smallrye/smallrye-reactive-utils#67]
currently, updating to Vert.x 3.8.1 fails with:

```
[ERROR] Tests run: 30, Failures: 8, Errors: 0, Skipped: 0, Time elapsed: 8.42 s <<< FAILURE! - in io.vertx.axle.test.FlowableReadStreamAdapterBackPressureTest
[ERROR] testEndWhenPaused(io.vertx.axle.test.FlowableReadStreamAdapterBackPressureTest)  Time elapsed: 1.068 s  <<< FAILURE!
java.lang.AssertionError: Was expecting at least one event

[ERROR] testDeliverEndWhenPaused(io.vertx.axle.test.FlowableReadStreamAdapterBackPressureTest)  Time elapsed: 1.023 s  <<< FAILURE!
java.lang.AssertionError: Was expecting at least one event

[ERROR] testFoo(io.vertx.axle.test.FlowableReadStreamAdapterBackPressureTest)  Time elapsed: 1.013 s  <<< FAILURE!
java.lang.AssertionError: Was expecting at least one event

[ERROR] testEndDuringResume(io.vertx.axle.test.FlowableReadStreamAdapterBackPressureTest)  Time elapsed: 1.012 s  <<< FAILURE!
java.lang.AssertionError: Was expecting at least one event

[ERROR] testEndDuringRequestResume(io.vertx.axle.test.FlowableReadStreamAdapterBackPressureTest)  Time elapsed: 1.013 s  <<< FAILURE!
java.lang.AssertionError: Was expecting at least one event

[ERROR] testBackPressureBuffer(io.vertx.axle.test.FlowableReadStreamAdapterBackPressureTest)  Time elapsed: 1.009 s  <<< FAILURE!
java.lang.AssertionError: Was expecting at least one event

[ERROR] testChained(io.vertx.axle.test.FlowableReadStreamAdapterBackPressureTest)  Time elapsed: 1.01 s  <<< FAILURE!
java.lang.AssertionError: Was expecting at least one event

[ERROR] testEndWithoutRequest(io.vertx.axle.test.FlowableReadStreamAdapterBackPressureTest)  Time elapsed: 1.012 s  <<< FAILURE!
java.lang.AssertionError: Was expecting at least one event

[INFO]
[INFO] Results:
[INFO]
[ERROR] Failures:
[ERROR]   FlowableReadStreamAdapterBackPressureTest>ReadStreamAdapterBackPressureTest.testBackPressureBuffer:366 Was expecting at least one event
[ERROR]   FlowableReadStreamAdapterBackPressureTest>ReadStreamAdapterBackPressureTest.testChained:380 Was expecting at least one event
[ERROR]   FlowableReadStreamAdapterBackPressureTest>ReadStreamAdapterBackPressureTest.testDeliverEndWhenPaused:142->ReadStreamAdapterBackPressureTest.testDeliverEndOrFailWhenPaused:165 Was expecting at least one event
[ERROR]   FlowableReadStreamAdapterBackPressureTest>ReadStreamAdapterBackPressureTest.testEndDuringRequestResume:137 Was expecting at least one event
[ERROR]   FlowableReadStreamAdapterBackPressureTest>ReadStreamAdapterBackPressureTest.testEndDuringResume:255 Was expecting at least one event
[ERROR]   FlowableReadStreamAdapterBackPressureTest>ReadStreamAdapterBackPressureTest.testEndWhenPaused:174->ReadStreamAdapterBackPressureTest.testEndOrFailWhenPaused:196 Was expecting at least one event
[ERROR]   FlowableReadStreamAdapterBackPressureTest>ReadStreamAdapterBackPressureTest.testEndWithoutRequest:85->ReadStreamAdapterBackPressureTest.testEndOrFailWithoutRequest:100 Was expecting at least one event
[ERROR]   FlowableReadStreamAdapterBackPressureTest>ReadStreamAdapterBackPressureTest.testFoo:279 Was expecting at least one event
[INFO]
[ERROR] Tests run: 62, Failures: 8, Errors: 0, Skipped: 0
```

**Comment by vietj** [comment]
see https://github.com/vert-x3/vertx-rx/commit/323c49056f93d8facd242ced7cc60ccdfe03d689

**Comment by cescoffier** [comment]
Fixed in #68
